datasets/HakaiADCPTimeSeriesProfileProvisional.xml

```diff
@@ -26,7 +26,11 @@
 		<att name="Conventions">CF-1.8, COARDS, ACDD-1.3</att>
 		<att name="date_modified">2020-10-09T13:41:01Z</att>
 		<att name="infoUrl">https://www.hakai.org</att>
-		<att name="institution">Hakai Institute,University Of Victoria</att>
+		<att name="institution">Hakai Institute</att>
+		<att name="creator_name">Hakai Institute</att>
+		<att name="creator_type">institution</att>
+		<att name="contributor_name">Jessy Barrette [Hakai Institute];Pramod Thupaki [Hakai Institute]; Brian Timmer [University of Victoria]</att>
```

Comment:
yeah ok that, that contributor_field is open text. I picked ";" because it had the least chances to be used within the names, but commas would be fine if you prefer

Reply:
commas and regular brackets would be more consistent with the other fields, eg
`Jessy Barrette (Hakai Institute), Pramod Thupaki (Hakai Institute), Brian Timmer (University of Victoria)`

Reply:
ok